modules/ROOT/pages/bolt/message.adoc

```diff
@@ -720,19 +721,33 @@ FAILURE {'code': 'Neo.ClientError.Request.Invalid', 'message': "Message 'LogoffM
 === Request message `TELEMETRY`
 
 The `TELEMETRY` message contains an integer representing which driver API was used.
+
+This message is new in version *5.4*.
```

Comment:
We can also add `label:new[Introduced in 5.4]` beside heading to have a label show up. It looks much nicer, but it would also be nice to have them throughout the whole docset. We can have a conversation on this.

![Screenshot from 2023-10-06 08-29-38](https://github.com/neo4j/docs-bolt/assets/114478074/9a5599fe-e89a-454b-8999-3c9ab7ca359c)

Reply:
I really like this tag. But I'd go for consistency. Can these labels only be applied to headlines? Because there are also fields and details that changed withing messages.

Reply:
They can be applied to any element (paragraphs, table entries, etc), they get attached to its nearest asciidoc element (i.e. you can write
```
== Section header
label:new[Introduced in 5.4]
```
and the label still belongs to the header).

Reply:
I don't have a strong opinion over whether this should happen here or later in the whole manual. I just wanted to point it out :)

Reply:
I created an issue (#44) to take care of that in a separate PR.